main.ts

```diff
@@ -55,9 +65,53 @@ class TerraformCdkProviderStack extends TerraformStack {
       );
     }
 
-    const team = new DataGithubTeam(this, "cdktf-team", {
+    const githubProviderHashiCorp = new GithubProvider(
+      this,
+      "github-provider-hashicorp",
+      {
+        owner: "hashicorp",
+      }
+    );
+
+    const githubProviderCdktf = new GithubProvider(
+      this,
+      "github-provider-cdktf",
+      {
+        owner: "cdktf",
+        alias: "cdktf",
```

Comment:
This is interesting. Since we're using the variable everywhere, we 'technically' don't need to provider the `alias`. However, if we don't, terraform will get confused / fail, right? 

If so, it's often hard for newcomers to figure out these kinds of issues coming from optional arguments.

Reply:
This comment and the one above are both irrelevant for this PR, but it made me wonder: what does the `name` in the provider do? Since Terraform doesn't have that concept:

```tf
provider "aws" {
  alias = "test"
}
```

Reply:
What `name` do you mean? The construct id?

Reply:
> What `name` do you mean? The construct id?

Yeah. The second argument: `github-provider-cdktf`

Reply:
That's because every construct needs to have an id.